ui/push-health/ClassificationGroup.jsx

```diff
@@ -97,50 +116,99 @@ class ClassificationGroup extends React.PureComponent {
         </h4>
         <Collapse isOpen={detailsShowing} className="w-100">
           {hasRetriggerAll && Object.keys(group).length > 0 && (
-            <ButtonGroup>
-              <Button
-                title="Retrigger all 'Need Investigation' jobs once"
-                onClick={() => this.retriggerAll(1)}
-              >
-                <FontAwesomeIcon
-                  icon={faRedo}
-                  title="Retrigger"
-                  className="mr-2"
-                />
-                Retrigger all
-              </Button>
-              <ButtonDropdown
-                isOpen={retriggerDropdownOpen}
-                toggle={this.toggleRetrigger}
-              >
-                <DropdownToggle caret />
-                <DropdownMenu>
-                  {[5, 10, 15].map(times => (
-                    <DropdownItem
-                      key={times}
-                      title={`Retrigger all 'Need Investigation' jobs ${times} times`}
-                      onClick={() => this.retriggerAll(times)}
+            <Navbar className="mb-4">
+              <Nav>
+                <NavItem>
+                  <ButtonGroup size="sm">
+                    <Button
+                      title="Retrigger all 'Need Investigation' jobs once"
+                      onClick={() => this.retriggerAll(1)}
+                      size="sm"
                     >
-                      Retrigger all {times} times
-                    </DropdownItem>
-                  ))}
-                </DropdownMenu>
-              </ButtonDropdown>
-            </ButtonGroup>
+                      <FontAwesomeIcon
+                        icon={faRedo}
+                        title="Retrigger"
+                        className="mr-2"
+                      />
+                      Retrigger all
+                    </Button>
+                    <ButtonDropdown
+                      isOpen={retriggerDropdownOpen}
+                      toggle={this.toggleRetrigger}
+                      size="sm"
+                    >
+                      <DropdownToggle caret />
+                      <DropdownMenu>
+                        {[5, 10, 15].map(times => (
+                          <DropdownItem
+                            key={times}
+                            title={`Retrigger all 'Need Investigation' jobs ${times} times`}
+                            onClick={() => this.retriggerAll(times)}
+                          >
```

Comment:
I think we need to start adding `tag="a"` to the `DropdownItem` for accessibility. Is this correct @yogmel?